### PR TITLE
Database init performance adjustments & cleanups

### PR DESCRIPTION
**Pull request by lsipii**
- drop the database seeds, database warmup operations
  - initial usage seems to have been to warm up the serverless database (for cost reasons), which is not the responsibility of the app but of the infra
- disabled the automatic database migration procedure from the live-environments (dev, staging, production)
  - keep enabled on local-env 
- create strategy for the live environment database migrations: 
  - migrate on CI/CD-deployment when needed
  - include pre-deploy database backup mechanism
  - include documentation for reverting the database to a backup snapshot with pulumi
 - refactor Github Action workflows for more reusable setup
 - fix several dotnet warnings
 - flatten the database migrations history
   - needs manual steps on pre-deployment to existing db-setup

### CODE REVIEW

**Comment by lsipii** [comment]
https://mabster.net/posts/ef-migration-github-actions/

**Review by lassipatanen — APPROVED** [review]
Vaikuttaa ihan pätevältä ratkaisulta 👍

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  build-and-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 6.0.x
+      - name: Test and build
+        run: make prep-deploy
```
[the remark this answers is not in the document]
Comment (lsipii) [reply .github/workflows/build-and-test.yml:16]: Lokaalista on mahdollista tehdä pulumilla deployment ja sitä varten on kätevää että kaikki itse buildaamiseen tarvittavat stepit on määritelty jossain yksittäisesssä paikassa, että ne on ajopaikasta riippumatta samat stepit mitä ajetaan.

[the remark this answers is not in the document]
Comment (lassipatanen) [reply .github/workflows/build-and-test.yml:16]: rgr 👍

**Pull request merged**